pgmetadata/classes/pgmetadataProfile.class.php

```diff
@@ -11,14 +11,15 @@
 class pgmetadataProfile {
 
     /**
-    * Get the Name of the cadastre DB profile
+    * Get the Name of the pgmetadata DB profile
     * @param project Project key
     * @param repository Repository key
     * @param layerName Name of the Parcelle layer
     * @param profile The default cadastre DB profile
     * @return Name of the cadastre DB profile
```

Comment:
`Parcelle`, `Cadastre` ...

I think you should do a `grep -rni "parcelle" *` and also with `cadastre`, `adresse` etc
Use generic words